apps/researcher/src/components/add-object-list-form/index.tsx

```diff
@@ -0,0 +1,151 @@
+'use client';
+
+import {useAuth} from '@clerk/nextjs';
+import {addList} from './actions';
+import {useForm, SubmitHandler} from 'react-hook-form';
+import {zodResolver} from '@hookform/resolvers/zod';
+import {insertObjectListSchema} from '@colonial-collections/database/src/db/schema';
```

Comment:
Is there a more 'elegant'/abstract way to import from the `database` package? It now contains the full path to a specific place in the `src` folder. Would something like this be possible?

`import {insertObjectListSchema} from '@colonial-collections/database';`